docs/Timers/index.md

```diff
@@ -0,0 +1,155 @@
+# Timers
+
+Why should you use these timers over the countless options avaliable in vanilla? Performance and reliability.
+
+There are three issues with vanilla timers, they are:
+
+- The param function arguments are nulled after the first call
+- Timer running and iteration is based soley on function calls and dynamic arrays, these are expensive instructions in Enforce Script
+- The creation and management of timers can be convuluted. They were designed for UI and delayed calls. 
+
+There is one major issue with `ScriptCallQueue`, after a server has been up for a prolonged time there will be a dramatic loss of server performance, as evident from [Game Labs](). This is not just from modders forgetting to clean up their timers.
+
+The timers Community Framework is providing will offer improved performance by removing unneccessary function calls and removing the use dynamic arrays. As testing has shown with [Expression's](../ExpressionVM/index.md) there is a huge performance gain when using dual Linked Lists over even static arrays. 
+
+All timers have safeguards built in, such as with [Instance Timers](#Instance-Timers) only allowing to be ran in `Managed` classes. 
+
+## Instance Timers
+
+Instance Timers can only be ran in `Managed` classes. This benefit provides safe memory management and allows for timers to be deleted when they are no longer required.
+
+### Basic Example
+
+```csharp
+class SomeClass : Managed
+{
+    void SomeClass()
+    {
+        float interval = 0.5; // The timer will tick every 0.5 seconds.
+        CF_Timer.Create(this, "OnTick", interval);
+    }
+
+    void OnTick(CF_TimerBase timer, float deltaTime)
+    {
+        Print("Timer has ran for " + timer.m_TimerElapsed + " seconds.");
```

Comment:
You said in DM's to encourage the use of member variables

Reply:
I did? Can't remember to have said that, and especially not for a context like that.
Especially with something like this, the inner workings should not be exposed. See my comment on protecting the members to avoid setting it from external code.